ckanext/spatial/harvesters/base.py

```diff
@@ -33,6 +34,11 @@
 from ckanext.spatial.model import ISODocument
 from ckanext.spatial.interfaces import ISpatialHarvester
 
+if p.toolkit.check_ckan_version("2.9"):
```

Comment:
How about `min_version="2.9"`, to be compatible with 3.0 when it comes out?

Reply:
This improvement could go into a second PR - there's no point delaying merging this PR.